content/en/docs/Responses/ReplayGain.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "ReplayGain"
+linkTitle: "ReplayGain [OS]"
+opensubsonic:
+- Addition
+description: >
+  The replay gain data of a song.
+---
+
+{{< tabpane persistLang=false >}}
+{{< tab header="**Example**:" disabled=true />}}
+{{< tab header="OpenSubsonic" lang="json">}}
+{
+    "trackGain": 0.1,
+    "albumGain": 1.1,
```

Comment:
I have encountered cases where the album gain was actually the disc gain (different gain for each disc of the same album)
Technically, this is no big deal here since ReplayGain is attached to a song, so "albumGain" would not be the same for all songs in the album.